package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "7.0.49",
+  "version": "7.1.0",
```

Comment:
I'm going to say that all changes up to 7.0.49 were "patches" to whatever we were working on before, and now this ticket a new topic (restoration stuff).

Lmk if you think I should increment to v7.0.50 instead.

Reply:
I think 7.1.0 is better